src/primitives/triangle/mod.rs

```diff
@@ -144,39 +138,37 @@ impl ContainsPoint for Triangle {
 
 impl Dimensions for Triangle {
     fn bounding_box(&self) -> Rectangle {
-        let x_min = min(min(self.p1.x, self.p2.x), self.p3.x);
-        let y_min = min(min(self.p1.y, self.p2.y), self.p3.y);
+        let [p1, p2, p3] = self.vertices;
 
-        let x_max = max(max(self.p1.x, self.p2.x), self.p3.x);
-        let y_max = max(max(self.p1.y, self.p2.y), self.p3.y);
+        let x_min = min(min(p1.x, p2.x), p3.x);
+        let y_min = min(min(p1.y, p2.y), p3.y);
+
+        let x_max = max(max(p1.x, p2.x), p3.x);
+        let y_max = max(max(p1.y, p2.y), p3.y);
 
         Rectangle::with_corners(Point::new(x_min, y_min), Point::new(x_max, y_max))
     }
 }
 
 impl Triangle {
-    /// Create a new triangle with a given style
+    /// Create a new triangle with the given vertices.
     pub const fn new(p1: Point, p2: Point, p3: Point) -> Self {
```

Comment:
This might be better if someone uses code completion to show the arguments of the function:
```suggestion
    pub const fn new(vertex1: Point, vertex2: Point, vertex3: Point) -> Self {
```
Don't worry I don't expect you to change all `p1`...`p3`s in the file :wink:.

Reply:
How about `vN`, etc, or is that too ambiguous compared to `vertexN` do you think?

Reply:
Is there a reason why you want to keep the names short? I just prefer more descriptive names in the public API. But I'm also OK with using `vN` if you prefer that.

Reply:
No that's a good point actually. I wanted to keep them consistent with `pN` used internally, but didn't consider these args are part of the public API. I've changed them to `vertexN`.